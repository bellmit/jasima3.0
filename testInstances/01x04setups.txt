# 1 machines, 4 routes, 4 jobs. see Pinedo pp. 341, 384 (example 14.2.1)

#number of machines
1

#number of routes
4

# machine orders <machine number(s)>;<setup state> (no space after or before comma/semicolon)
1;S1.1
1;S1.2
1;S1.3
1;S1.4

# processing times for each operation on a route
13
9
13
10


#setup time matrix for machine(s)
1

setup

# number of setup states (including DEF_SETUP) 
5

#setup from (row), into (col)
DEF_SETUP	0	1	1	3	4
S1.1		0	x	4	1	3
S1.2		0	0	x	1	0
S1.3		0	1	2	x	3
S1.4		0	4	3	1	x


# job specifications, jobs have to be ordered by release date
#
jobs

#number of jobs
4

# <route number> <release date> <due date> <weight>
1	0	12	2
2	0	37	4
3	0	21	2
4	0	22	5

