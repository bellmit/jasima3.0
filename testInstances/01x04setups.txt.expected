testInstances/01x04setups.txt SPT	simtime	51.0
testInstances/01x04setups.txt SPT	cMax	51.0
testInstances/01x04setups.txt SPT	flowtimes	29.50	10.00	51.00	18.16	118.00	4
testInstances/01x04setups.txt SPT	tardiness	13.75	0.00	30.00	16.01	55.00	4
testInstances/01x04setups.txt SPT	lateness	6.50	-27.00	30.00	26.39	26.00	4
testInstances/01x04setups.txt FCFS	simtime	54.0
testInstances/01x04setups.txt FCFS	cMax	54.0
testInstances/01x04setups.txt FCFS	flowtimes	34.00	14.00	54.00	17.30	136.00	4
testInstances/01x04setups.txt FCFS	tardiness	13.50	0.00	32.00	15.26	54.00	4
testInstances/01x04setups.txt FCFS	lateness	11.00	-10.00	32.00	18.65	44.00	4
testInstances/01x04setups.txt 	simtime	54.0
testInstances/01x04setups.txt 	cMax	54.0
testInstances/01x04setups.txt 	flowtimes	34.00	14.00	54.00	17.30	136.00	4
testInstances/01x04setups.txt 	tardiness	13.50	0.00	32.00	15.26	54.00	4
testInstances/01x04setups.txt 	lateness	11.00	-10.00	32.00	18.65	44.00	4
testInstances/01x04setups.txt 	simtime	51.0
testInstances/01x04setups.txt 	cMax	51.0
testInstances/01x04setups.txt 	flowtimes	29.50	10.00	51.00	18.16	118.00	4
testInstances/01x04setups.txt 	tardiness	13.75	0.00	30.00	16.01	55.00	4
testInstances/01x04setups.txt 	lateness	6.50	-27.00	30.00	26.39	26.00	4
testInstances/01x04setups.txt SST	simtime	49.0
testInstances/01x04setups.txt SST	cMax	49.0
testInstances/01x04setups.txt SST	flowtimes	32.50	14.00	49.00	15.02	130.00	4
testInstances/01x04setups.txt SST	tardiness	9.50	2.00	27.00	11.90	38.00	4
testInstances/01x04setups.txt SST	lateness	9.50	2.00	27.00	11.90	38.00	4
